### PR TITLE
removing non-ascii character

### PR DESCRIPTION
**Pull request by mabhard**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -5483,7 +5483,7 @@
           "items": {
             "type": "string"
           },
-          "description": "Availability Zone to use for this host group � only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
+          "description": "Availability Zone to use for this host group only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
```
Comment (celikcigdem) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Can we add a period after "Availability Zone to use for this host group" and make "Only single zone is supported" as a separate statement.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -5504,7 +5504,7 @@
...
```
Comment (celikcigdem) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: same here

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (mabhard) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Done

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 2 above)
Comment (mabhard) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Done

**Comment by celikcigdem** [comment]
Looks good to me. Signing off as the CRP SME for dedicated hosts.

**Review by zikalino — APPROVED** [review]

**Pull request merged**